src/windowsdesktop/eng/Versions.props

```diff
@@ -32,5 +32,12 @@
     <SystemSecurityPrincipalWindowsVersion>5.0.0</SystemSecurityPrincipalWindowsVersion>
     <!-- wcf -->
     <SystemServiceModelVersion>8.1.2</SystemServiceModelVersion>
+    <!-- WiX -->
+    <MicrosoftWixToolsetSdkVersion Condition="'$(MicrosoftWixToolsetSdkVersion)' == '' and '$(DefaultWixSdkVersion)' != ''">$(DefaultWixSdkVersion)</MicrosoftWixToolsetSdkVersion>
+    
+    <!-- Runtime version variables for CI builds (missing variables causing CI issues) -->
+    <DotNetRuntimeVersion>$(MicrosoftNETCoreAppRefVersion)</DotNetRuntimeVersion>
```

Comment:
We shouldn't use proxy properties for these. It ends up confusing with possibly weird behavior due to import orders.